src/Titon/Context/Definition/DefinitionFactory.hh

```diff
@@ -7,10 +7,10 @@
 
 namespace Titon\Context\Definition;
 
-use Closure;
 use Titon\Context\Depository;
 use Titon\Context\Definition;
 use Titon\Context\Exception\ItemNotDefinableException;
+use Closure;
```

Comment:
Yeah, I ran into this issue too :-P. Maybe we should put a comment saying this `use` is required otherwise it assumes Titon\Context\Closure?